tests/validators/test_typed_dict.py

```diff
@@ -1192,13 +1192,55 @@ class MyDataclass:
         }
     )
 
-    assert v.validate_python(Foobar()) == ({'a': 1, 'b': 2, 'c': 'ham'}, {'a', 'b', 'c'})
-    assert v.validate_python(MyDataclass()) == ({'a': 1, 'b': 2, 'c': 'ham'}, {'a', 'b', 'c'})
-    assert v.validate_python(Cls(a=1, b=2, c='ham')) == ({'a': 1, 'b': 2, 'c': 'ham'}, {'a', 'b', 'c'})
-    assert v.validate_python(Cls(a=1, b=datetime(2000, 1, 1))) == ({'a': 1, 'b': datetime(2000, 1, 1)}, {'a', 'b'})
+    assert v.validate_python(Foobar()) == ({'a': 1}, {'a'})
+    assert v.validate_python(MyDataclass()) == ({'a': 1}, {'a'})
+    assert v.validate_python(Cls(a=1, b=2, c='ham')) == ({'a': 1}, {'a'})
+    assert v.validate_python(Cls(a=1, b=datetime(2000, 1, 1))) == ({'a': 1}, {'a'})
```

Comment:
Before this change we were grabbing anything off of the class that is not a `function` type